examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -30,9 +31,18 @@ using Oceananigans.Units: minute, minutes, hour
 # ## The grid
 #
 # We use 32³ grid points with 2 m grid spacing in the horizontal and
-# 1 m spacing in the vertical,
+# varying spacing in the vertical, with higher resolution closer to the
+# surface
+
+S = 1.1 # stretching factor
+Nz = 24
+Lz = 32
+hyperbolically_spaced_faces(k) = Lz*(-1 + (1 + tanh(S * ( (k - 1) / Nz - 1)) / tanh(S)))
+grid = VerticallyStretchedRectilinearGrid(size=(32, 32, Nz), 
+                                          x=(0, 64), y=(0, 64), z_faces=hyperbolically_spaced_faces,
+                                          halo=(3,3,3),
+                                          )
```

Comment:
```suggestion
σ = 1.1 # stretching factor
Nz = 24
Lz = 32

hyperbolically_spaced_faces(k) = - Lz * (1 - (1 + tanh(σ * (k - 1) / Nz - 1) / tanh(σ)))

grid = VerticallyStretchedRectilinearGrid(size = (32, 32, Nz), 
                                          x = (0, 64),
                                          y = (0, 64),
                                          z_faces = hyperbolically_spaced_faces)
```

Just because I think it's important to set a good example regarding formatting. Also, `S` is a potentially confusing name for the stretching factor since salinity (denoted with `S`) is a variable in this problem.

Reply:
I was sure I had formatted it correctly, idk why it's formatted weirdly. But good point about using `S`.